src/components/new-request/business-lookup.vue

```diff
@@ -1,28 +1,31 @@
 <template>
   <!-- once in Summary state, need to re-mount to reuse this component -->
-  <div id="business-lookup" v-if="state !== States.SUMMARY">
+  <div
+    v-if="state !== States.SUMMARY"
+    id="business-lookup"
+  >
     <v-autocomplete
+      v-model="searchField"
```

Comment:
ESLint changed this and then I fixed it manually as per [similar code here](https://github.com/bcgov/bcrs-shared-components/blob/14b5f1e234826a04fb619588acbc05f058b935f6/src/components/business-lookup/BusinessLookup.vue#L5).

I will test this before merging.

Reply:
I broke this. I'm working on a fix now.

Reply:
I reverted the code change and added an ESLint rule exception instead.

I tested this and the business lookup works as before.